Framework/AODMerger/src/aodThinner.cxx

```diff
@@ -194,22 +200,38 @@ int main(int argc, char* argv[])
       exitCode = 8;
       break;
     }
+    // TrackQA
+    TTree* trackQA;
+    if (hasTrackQA && (trackQA = (TTree*)inputFile->Get(Form("%s/%s", dfName, trackQAName.Data()))) == nullptr) {
+      exitCode = 20;
+      break;
+    }
 
     // We need to loop over the V0s once and flag the prong indices
     int trackIdxPos = 0, trackIdxNeg = 0;
-    std::unordered_map<int, bool> keepV0TPCs;
+    std::vector<bool> keepV0s(trackExtraTree->GetEntries(), false);
     v0s->SetBranchAddress("fIndexTracks_Pos", &trackIdxPos);
     v0s->SetBranchAddress("fIndexTracks_Neg", &trackIdxNeg);
-    auto nV0s = v0s->GetEntriesFast();
+    auto nV0s = v0s->GetEntries();
```

Comment:
Why this change (also elsewhere...)?

Reply:
Remnant of me thinking first lets use `GetEntriesFast` its faster, then realising that `GetEnries` and `GetEntriesFast` are equivalent in newer ROOT version and changing everything to the same. If not wanted can drop it?